edx_proctoring/static/proctoring/js/proctored_app.js

```diff
@@ -5,4 +5,16 @@ $(function() {
         model: new ProctoredExamModel()
     });
     proctored_exam_view.render();
+
+    // show the timer bar when the window scrolls.
+    $(window).scroll(function() {
+        if ($(this).scrollTop() > 60){
+            $(".proctored_exam_status").addClass('fixed');
+            $(".wrapper-course-material").css('margin-top', '58px');
```

Comment:
by adding that line `$(".wrapper-course-material").css('margin-top', '58px');`
<img width="1401" alt="screen shot 2015-08-19 at 7 28 12 pm" src="https://cloud.githubusercontent.com/assets/7291787/9359005/5be7eaec-46a8-11e5-855e-adbc4e2d7a1b.png">